test-e2e/test_calico_networking.py

```diff
@@ -37,6 +37,37 @@ def assert_system_unit_state(node: Node, unit_name: str, active: bool = True) ->
         assert "ConditionResult=no" in unit_properties
 
 
+def test_calico_disabled(docker_backend: Docker, artifact_path: Path,
```

Comment:
Can we skip this test using `only_changed` unless the Calico package is modified?

Reply:
Yep I'll whip that up in a new pr quick since this one is in the train.